modules/ServerAPI/src/client/Protocol.coffee

```diff
@@ -290,6 +320,7 @@ class ProtocolBaseController extends BaseEntityController
 			@$('.bv_creationDate').val UtilityFunctions::convertMSToYMDDate(@model.getCreationDate().get('dateValue'))
 		@$('.bv_assayTreeRule').val @model.getAssayTreeRule().get('stringValue')
 		@$('.bv_assayPrinciple').val @model.getAssayPrinciple().get('clobValue')
+		@$('.bv_strictEndpointMatchingInputCheckbox').prop("checked", @model.getStrictEndpointMatching().get('codeValue'));
```

Comment:
When the page is loaded, find out if the protocol has strict endpoint matching and check the box (or leave the box unchecked) accordingly.